sztp-agent/pkg/dhcp/network_manager_test.go

```diff
@@ -0,0 +1,8 @@
+package dhcp
```

Comment:
Since you opened new package for dhcp, let’s move dhcp file parsing functions here as well

Reply:
so move `getBootstrapURL()` here.
or jus the 'LinesInFileContains()'

Reply:
Yeah, both I think… wdyt?

Reply:
yep thought of the same